ui/admin/app/components/workflow/steps/While.tsx

```diff
@@ -51,7 +51,7 @@ export function WhileComponent({
 
     return (
         <div className={cn("border rounded-md", className)}>
-            <ButtonDiv
+            <ClickableDiv
```

Comment:
Oh cool, I should have used that

Reply:
Yeah I got annoyed that shadcn doesn't provide a camouflaged button